views/ddox.inc.symbol-search.dt

```diff
@@ -3,22 +3,9 @@ noscript
 
 #symbolSearchPane(style="display: none")
 	p
-		input#symbolSearch(type="text", placeholder="Search for symbols", onchange="performSymbolSearch(24);", onkeypress="this.onchange();", onpaste="this.onchange();", oninput="this.onchange();")
+		- if (info.settings.focusSearchField)
+			input#symbolSearch(type="text", placeholder="Search for symbols", onchange="performSymbolSearch(24);", onkeypress="this.onchange();", onpaste="this.onchange();", oninput="this.onchange();", autofocus)
+		- else
+			input#symbolSearch(type="text", placeholder="Search for symbols", onchange="performSymbolSearch(24);", onkeypress="this.onchange();", onpaste="this.onchange();", oninput="this.onchange();")
```

Comment:
You can also use a boolean attribute instead of the `if`:
```
input#symbolSearch(type="text", placeholder="Search for symbols", onchange="performSymbolSearch(24);", onkeypress="this.onchange();", onpaste="this.onchange();", oninput="this.onchange();", autofocus=info.settings.focusSearchField)
```